src/components/Label/tests/Label.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Label from '../Label';
+
+describe('Label Suit', () => {
+
+    it('Verify Label is not Hidden', () => {
```

Comment:
For each property, add a `describe('hidden property', () => {` and add child describe for following cases:
1. Not Set
2. Set to valid or true case.
3. Set to undefined or false case. 

Then write all it statements relevant to those cases. For more detail, review Chip Component.